.github/workflows/scale-test-gce.yaml

```diff
@@ -140,6 +143,7 @@ jobs:
             --control-plane-count 1 \
             --control-plane-size n2-standard-4 \
             --node-count 1 \
+            --node-size e2-standard-8 \
```

Comment:
Do you mind adding a comment explaining why we are using n2 for cp and e2 for worker nodes? A quick Google search explains it, but it'd be nice to have it in here too.

Reply:
done.